README.md

```diff
@@ -1,2 +1,73 @@
-# testing
-Utilities for testing front-end components and applications
+# @brightspace-ui/testing
+
+[![NPM version](https://img.shields.io/npm/v/@brightspace-ui/testing.svg)](https://www.npmjs.org/package/@brightspace-ui/testing)
+
+Testing utilities that build upon [@web/test-runner](https://modern-web.dev/docs/test-runner/overview/) but are specifically designed and configured to work with Brightspace components and applications.
+
+## Installation
+
+Install from NPM:
+
+```shell
+npm install @brightspace-ui/foo
+```
+
+## Developing, Testing and Contributing
+
+After cloning the repo, run `npm install` to install dependencies.
+
+### Linting
+
+```shell
+# eslint
+npm run lint
+```
+
+### Testing
+
+```shell
+# lint & run headless unit tests
+npm test
+
+# unit tests only
+npm run test:headless
+```
```

Comment:
We can add documentation for the actual functionality once it solidifies a bit more.